timescaledb/how-to-guides/analytics/approximate-percentile.md

```diff
@@ -0,0 +1,68 @@
+# Approximate percentiles
+In general, percentiles are useful for understanding the distribution of data.
+The 50th percentile is the point at which half of your data is greater and half
+is lesser. The 10th percentile is the point at which 90% of the data is greater,
+and 10% is lesser. The 99th percentile is the point at which 1% is greater, and
+99% is lesser.
+
+The 50th percentile, or median, is often a more useful measure than the average,
+especially when your data contains outliers. Outliers can dramatically change
+the average, but do not effect the median as much. For example, if you have
+three rooms in your house and two of them are 40℉ (4℃) and one is 130℉ (54℃),
+the fact that the average room is 70℉ (21℃) doesn't matter much. However, the
+50th percentile temperature is 40℉ (4℃), and tells you that at least half your
+rooms are at refrigerator temperatures (also, you should probably get your
+heating checked!)
+
+Percentiles are sometimes used less frequently because they can use more CPU and
+memory to calculate than an average or another aggregate measure. This is
+because an exact computation of the percentile needs the full dataset as an
+ordered list. Timescale Analytics uses approximation algorithms to calculate a
+percentile without requiring all of the data. This also makes them more
+compatible with continuous aggregates. By default, Timescale Analytics uses
+`uddsketch`, but you can also choose to use `tdigest`. See
+the [Analytics documentation][gh-analytics-algorithms] for more information
+about these algorithms.
+
+<highlight type="tip">
+Technically, a percentile divides a group into 100 equally sized pieces, while a
+quintile divides a group into an arbitrary number of pieces. Because we don't
+always use exactly 100 buckets, "quintile" is the more technically correct term
+in this case. However, we use the word "percentile" because it's a more common
+word for this type of function.
+</highlight>
+
+## Run an approximate percentage query
+In this procedure, we are using an example table called `response_times` that contains information about how long a server takes to respond to API calls.
+
+### Procedure: Running an approximate percentage query
+1.  At the `psql` prompt, create a continuous aggregate that computes the daily aggregates:
+    ```sql
+    CREATE MATERIALIZED VIEW response_times_daily
+    WITH (timescaledb.continuous)
+    AS SELECT
+      time_bucket('1 day'::interval, ts) as bucket,
+      percentile_agg(response_time_ms)
+    FROM response_times
+    GROUP BY 1;
+    ```
+1.  Re-aggregate the aggregate to get the last 30 days, and look for the 95th percentile:
+    ```sql
+    SELECT approx_percentile(0.95, percentile_agg(percentile_agg)) as threshold
```

Comment:
This has been modified to be rollup instead of percentile_agg(percentile_agg). That may have happened after you took it though.

Reply:
can you please propose the correct command?

Reply:
yep, this is a general thing and our docs have been updated to reflect it ie: 

```SQL
SELECT
    api_id,
    mean(rollup(percentile_agg)) as avg,
    approx_percentile(0.5, rollup(percentile_agg)) as approx_median
FROM response_times_hourly
GROUP BY api_id
ORDER BY api_id;
```

in general though for all of these whereever we see: `percentile_agg(percentile_agg)` -> `rollup(percentile_agg)`